opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -102,11 +102,35 @@ option go_package = "go.opentelemetry.io/proto/otlp/profiles/v1development";
 // as well.
 message ProfilesData {
   // An array of ResourceProfiles.
-  // For data coming from a single resource this array will typically contain
-  // one element. Intermediary nodes that receive data from multiple origins
-  // typically batch the data before forwarding further and in that case this
-  // array will contain multiple elements.
+  // A ResourceProfile will be created for each virtual environment. Any
+  // ScopeProfiles that are not assigned to a virtual environment should be
+  // collected within an additional ResourceProfile. Furthermore, a
+  // ResourceProfile will be created for each value assigned to the RESOURCE
+  // environment variable.
```

Comment:
This is a details of a theoretical producer. How does this apply here?

Reply:
Yeah, I think this comment is overfitting a bit to the eBPF profiler, and perhaps the mention of the `RESOURCE` env variable is going into implementation details that are better specified elsewhere.

Maybe something like this could work better:

```
An array of ResourceProfiles.
For data coming from an SDK profiler, this array will typically contain one element. Host-level
profilers will usually create on ResourceProfile per container, as well as one additional
ResourceProfile for samples from non-containerized processes. Other resource groupings are
possible as well and clarified via Resource.attributes and semantic conventions.
```

Reply:
Suggetions was applied with https://github.com/open-telemetry/opentelemetry-proto/pull/644/commits/bfbe49bb4a11e1e58cfb2b32fdf63f3d651f5f52.